ci/precise_benchmarks.py

```diff
@@ -0,0 +1,150 @@
+import subprocess
+import os
+import sys
+import json
+from time import gmtime, strftime
+import platform
+import git
+
+if(len(sys.argv) != 2):
+    print("Usage: ./precise_benchmarks.py <ABS PATH TO PROCESS>")
+    exit(1);
+
+def run(cmd):
+    proc = subprocess.Popen(cmd,
+        stdout = subprocess.PIPE,
+        stderr = subprocess.STDOUT,
+    )
+    stdout, stderr = proc.communicate()
+    return proc.returncode, stdout, stderr
+def get_colon_index(string):
+    index = 0
+    found = False
+    for char in string:
+     if char == ':':
+         Found = True
```

Comment:
This isn't changing the same variable referenced on lines 22 and 28. This looks like a bug.

However, the bug doesn't matter, because we return on the next line anyway. So what's the point of this assignment? Delete it.